global-cli/index.js

```diff
@@ -167,6 +170,29 @@ function checkNodeVersion() {
   }
 }
 
+function checkAppName(appName) {
+  // TODO: there should be a single place that holds the dependencies
+  var dependencies = ['react', 'react-dom'];
+  var devDependencies = ['react-scripts'];
+  var allDependencies = dependencies.concat(devDependencies).sort();
+
+  if (allDependencies.indexOf(appName) >= 0) {
+    console.error(
+      chalk.red(
+        `Can't use "${appName}" as the app name because a dependency with the same name exists.\n\n` +
```

Comment:
Also requirement for Node >=4 is stated in the README. It is misleading. Shall we change that?
![screen shot 2016-09-18 at 14 03 55](https://cloud.githubusercontent.com/assets/613647/18615506/dc45327a-7da8-11e6-8610-cb2701a35d4c.png)